tests/test_lint_makefile.py

```
@@ -3,17 +3,22 @@
 from make_profiler import parser, lint_makefile
 
 
-def test_missing_rule(capsys):
-    mk = "all: foo\n"
+def run_validation(mk: str):
```

Ruff (0.11.9)

Missing return type annotation for public function `run_validation`

(ANN201)